src/index.html

```diff
@@ -1,63 +1,85 @@
+<!DOCTYPE html>
 <html>
-<head>
-    <title>Daily Demo Boilerplate</title>
+  <head>
+    <title>Daily Prejoin Presence Demo</title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
-    <meta
+    <!--meta
       http-equiv="Content-Security-Policy"
       content="default-src 'self'; frame-src 'self' https://*.daily.co; script-src 'self' https://unpkg.com/@daily-co/daily-js; worker-src 'self';"
-    />
+    /-->
     <link rel="stylesheet" type="text/css" href="./css/style.css" />
     <link rel="stylesheet" type="text/css" href="./css/header.css" />
 
     <link rel="shortcut icon" href="./assets/favicon.ico" />
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="header-container">
-          <div class="left-header">
-            <img
-              src="./assets/daily.svg"
-              class="daily-logo"
-              alt="Daily logo reads Daily"
-            />
-            <strong>
-              <p class="header-title">Daily demo</p>
-            </strong>
-          </div>
-          <div class="right-header">
-            <a
-              href="https://docs.daily.co/reference"
-              target="_blank"
-              class="link"
-            >
-              <button class="button docs-button white-button">
-                API docs
-                <img
-                  class="arrow-icon"
-                  src="./assets/new-tab-icon.png"
-                  alt="Arrow icon points up and to the right"
-                  aria-hidden="true"
-                />
-              </button>
-            </a>
-            <span class="divider"></span>
-            <a href="https://github.com/daily-demos/boilerplate-vanilla-js/" target="_blank">
+      <div class="header-container">
+        <div class="left-header">
+          <img
+            src="./assets/daily.svg"
+            class="daily-logo"
+            alt="Daily logo reads Daily"
+          />
+          <strong>
+            <p class="header-title">Daily Prejoin Presence Demo</p>
+          </strong>
+        </div>
+        <div class="right-header">
+          <a
+            href="https://docs.daily.co/reference"
+            target="_blank"
+            class="link"
+          >
+            <button class="button docs-button white-button">
+              API docs
               <img
-                class="octocat"
-                src="./assets/github.png"
-                alt="GitHub logo Octocat"
+                class="arrow-icon"
+                src="./assets/new-tab-icon.png"
+                alt="Arrow icon points up and to the right"
+                aria-hidden="true"
               />
-            </a>
+            </button>

```

Comment:
Just clicked the button and got this error: `Error: You don't seem to have Go installed. Go to https://golang.org/doc/install for installation instructions.`
A note to make sure you have Go installed in the README would be good. (I know this is one of those things you don't notice when you're writing it because you're already set up for things to work. 😄 )

Reply:
whoops, put this comment on the wrong button. Should be on the create room button below.

Reply:
Ahh great catch, ty!